v12.0-prealpha/TF2-SDK release note (#11)
NeotokyoRebuild/NeotokyoRebuild.github.io: src/blog/2025-02-25/index.md

title: Pre-Alpha v12.0 Release Note - TF2 SDK update and now 64-bits!
summary: Release note for Neotokyo; Rebuild Pre-Alpha v12.0
author: nullsystem

# Pre-Alpha v12.0 Release Note - TF2 SDK update and now 64-bits!
2025-02-25 nullsystem

> * [Download v12.0-prealpha build](https://github.com/NeotokyoRebuild/neo/releases/tag/v12-prealpha)

Reviewer: Link typo, correct link: https://github.com/NeotokyoRebuild/neo/releases/tag/v12.0-prealpha

* [Install NT;RE (client)](/guide/install/)
* [GitHub Issues (Bug reports and feature requests)](https://github.com/NeotokyoRebuild/neo/issues)

## Source 2013 SDK Update - TF2 SDK Update and 64-bits

Valve last week released an update to the Source 2013 SDK also known as the [TF2 SDK update](https://www.teamfortress.com/post.php?id=238809)!
Technically we don't really use any TF2 functionalities, but the move to 64-bits and
10-years worth of changes and bugfixes makes this a very welcome major update for any
mods based on the engine.

NT;RE quickly got ported over in 5 days to the new update and is now here in

> v12.0-prealpha! Some notable extras is "Steam Networking" support (ability to hosts

Reviewer: Typos:
* `Some notable extras is` -> `Some notable extras are`
* `ability to hosts game` -> `ability to host a game`
* `choseable` -> `choosable`

game without a dedicated server), improved UI scaling from the engine, borderless window choseable
from the settings menu, some performance improvements, and more.

### Known Issues

However, it has also brought some issues. First, the distribution of the
dedicated server misplaced the 64-bits srcds into the client's directory,
and on Linux, a library and 64-bits version of the run script is missing.

> If you're running a dedicated server, you'll need to fetch some few files

Reviewer: Language nits:
* `fetch some few files` -> `fetch some files`
* `will just segfaults` -> `will just segfault`
* `have a regression` -> `has a regression`
* `that is not lowercase` -> `that are not lowercase`
* `showup` -> `show up`

title: Pre-Alpha v12.0 Release Note - TF2 SDK update and now 64-bits!
summary: Release note for Neotokyo; Rebuild Pre-Alpha v12.0
author: nullsystem

> # Pre-Alpha v12.0 Release Note - TF2 SDK update and now 64-bits!

Reviewer 2: ```suggestion
# Pre-Alpha v12.0 Release Notes - TF2 SDK update and now 64-bits!
```

2025-02-25 nullsystem (Proof-read by rainyan)

* [Download v12.0-prealpha build](https://github.com/NeotokyoRebuild/neo/releases/tag/v12.0-prealpha)
* [Install NT;RE (client)](/guide/install/)
* [GitHub Issues (Bug reports and feature requests)](https://github.com/NeotokyoRebuild/neo/issues)

## Source 2013 SDK Update - TF2 SDK Update and 64-bits

Valve last week released an update to the Source 2013 SDK also known as the [TF2 SDK update](https://www.teamfortress.com/post.php?id=238809)!

> Technically we don't really use any TF2 functionalities, but the move to 64-bits and

Reviewer 2: ```suggestion
Technically we don't really use any TF2 functionalities, but the upgrade to 64-bit and
```

> 10-years worth of changes and bugfixes makes this a very welcome major update for any

Reviewer 2: ```suggestion
10-years worth of changes/bugfixes makes this a very welcome major update for any
```

mods based on the engine.

NT;RE quickly got ported over in 5 days to the new update and is now here in

> v12.0-prealpha! Some notable extras are "Steam Networking" support (ability to host a

Reviewer 2: ```suggestion
v12.0-prealpha! Some notable extras are:
- "Steam Networking" support (ability to host a
```

> game without a dedicated server), improved UI scaling from the engine, borderless window choosable

Reviewer 2: ```suggestion
game without a dedicated server)
- Improved UI scaling from the engine
- Borderless window choosable
```

> from the settings menu, some performance improvements, and more.

Reviewer 2: ```suggestion
from the settings menu
- Some performance improvements, and more.
```

### Known Issues

However, it has also brought some issues. First, the distribution of the

> dedicated server misplaced the 64-bits srcds into the client's directory,

Reviewer 2: ```suggestion
dedicated server misplaced the 64-bit srcds binary into the client's directory,
```

> and on Linux, a library and 64-bits version of the run script is missing.

Reviewer 2: ```suggestion
and on Linux, a library and 64-bit version of the run script is missing.
```

If you're running a dedicated server, you'll need to fetch some files
from the client-side or even TF2's dedicated server. Another known issue,
the Linux dedicated server will just segfault on startup.

> The Linux client has a regression on picking up fonts with filenames

Reviewer 2: ```suggestion
The Linux client has some regression on picking up fonts with filenames
```
